### PR TITLE
LE1CoolDownWarmup+DefnEdit

### PR DESCRIPTION
**Pull request by jkostiuk**
Further address #370

Here is my suggestion for a warmup and cooldown for LE1. The warmup is basically just addressing the geometry in the RAP; the cooldown has them think a little more critically about the notational construct of augmented matrix. 

I also took the opportunity while I was in LE1 to splice in a definition of coefficient matrix to Definition 1.1.16--let me know what you think.

### CODE REVIEW

File: source/01-LE/01.ptx [hunk 1]
```diff
@@ -520,6 +568,15 @@ All inconsistent linear systems contain a logical <term>contradiction</term>.
       a_{m1} &amp; a_{m2} &amp; \cdots &amp; a_{mn} &amp; b_m
     \end{array}\right]
             </me>
+            Often, we will want to refer only to the coefficients of the linear system; the <term>coefficient matrix</term><idx>coeefficient matrix</idx> is the <m>m\times n</m> array consisting of these coefficients:
```
Comment (StevenClontz) [inline source/01-LE/01.ptx:571]:
```suggestion
            Eventually, we will find it useful to refer only to the coefficients of the linear system
            (and ignore its constant terms).
            We call the <m>m\times n</m> array consisting of these coefficients 
            a <term>coefficient matrix</term><idx>coeefficient matrix</idx>.
```

**Review by StevenClontz — CHANGES_REQUESTED** [review]
Consider my suggestion above.

File: source/01-LE/01.ptx (hunk 1 above)
Comment (jkostiuk) [reply source/01-LE/01.ptx:571]: hmmm...I think I'd prefer "Sometimes" to "Eventually", as minor as it may seem.

Comment (StevenClontz) [reply source/01-LE/01.ptx:571]: i've updated my suggestion accordingly

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**